plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout-block.spec.js

```diff
@@ -690,10 +685,12 @@ test.describe( 'Checkout Block page', () => {
 			.fill( customer.password );
 		await page.locator( 'text=Log in' ).click();
 		await expect(
-			page.getByRole( 'heading', { name: checkoutBlockPageTitle } )
+			page.getByRole( 'heading', { name: testPage.title } )
 		).toBeVisible();
 
 		// if edit address is present click it, otherwise fill shipping details
+		// eslint-disable-next-line playwright/no-networkidle
+		await page.waitForLoadState( 'networkidle' ); // need to wait for the page to load
```

Comment:
From watching this, it looks like the page loads and then an xhr request fires to retrieve/update the user & address section here.

We could do an API call on the customer to see if there's an address present and base it off of that.  I actually suspect that this might be a relic of the time we were running our tests in parallel on a single instance -- because I've never observed there NOT being an address (whereas before a test could have been running that updated the customer to not have an address in parallel).

Reply:
Just removing the conditional doesn't work.  It seems that this user doesn't have an address set up by default. So I'll keep going....

Reply:
Done!